Move packrat info to the Package Management page (#55)
moj-analytical-services/user-guidance: source/documentation/rshiny-app.md

By default, the template contains `server.R` and `ui.R` files. However, you may wish to take a different approach depending on your requirements. For example, using `app.R`, it is possible to deploy RShiny apps from within a package, as in this [example from the costmodelr package](https://github.com/RobinL/costmodelr/blob/b328902026bd1cce5d17b487e310c59725ea4d62/R/shiny_explorer.r#L20).

### List your package dependencies

> Most apps will have dependencies on various third-party packages (e.g., `dplyr`).

Reviewer: ```suggestion
Most apps will have dependencies on various third-party packages (for example, `dplyr`).
```

> In your project repo you will need a file that lists your dependencies. We use 'package management' to management these lists. The packages change through time and may not always be backwards compatible. To avoid compatibility issues and ensure reproducible outputs, it is necessary to use a package management system, such as conda, packrat or renv. For further guidance see: [package management](tools.html#managing-your-analytical-tools).

Reviewer: ```suggestion
In your project repository you will need a file that lists your dependencies. We use 'package management' to manage these lists. The packages change through time and may not always be backwards compatible. To avoid compatibility issues and ensure reproducible outputs, it is necessary to use a package management system, such as conda, packrat or renv. For further guidance see the [package management](tools.html#managing-your-analytical-tools) section.
```

> If using conda, you will need to [list your dependencies in your project's environment.yml](tools.html#exporting-your-environment).

Reviewer: ```suggestion
If using conda, you will need to [export your dependencies](tools.html#exporting-your-environment) to an `environment.yml` file.
```

> If using packrat, you will need to [list your dependencies in your project's packrat/packrat.lock](tools.html#packrat-usage).

Reviewer: ```suggestion
If using packrat, you will need to [export your dependencies](tools.html#packrat-usage) to a `packrat/packrat.lock` file.
```

> (During the Concourse build of an app, the app repo's Dockerfile has a command to install packages in the list.)

Reviewer: ```suggestion
The Dockerfile in the app's repository contains a command to install the packages in the list. This command will be run when Concourse builds the app.
```
